src/interfaces.ts

```diff
@@ -149,8 +149,7 @@ export interface QueryParams {
   fields?: Array<string>;
   offset?: number;
   limit?: number;
-  pageNumber?: number;
-  pageSize?: number;
+  page?: { [key: string]: string };
```

Comment:
we should deprecate offset/limit in favor of page I guess? But not yet remove since it is quite wildely used.

Reply:
how also about something like:


`
export interface QueryPageParams {
  [id: string]: string;
  offset?: string
  limit?: string;
  number?: string;
  size?: string;
}
`

or maybe allow string and number?